butler.py

```diff
@@ -157,7 +157,7 @@ def main():
   parser_deploy.add_argument(
       '--prod', action='store_true', help='Deploy to production.')
   parser_deploy.add_argument(
-      '--targets', nargs='*', default=['appengine', 'zips'])
+      '--targets', nargs='*', default=['appengine', 'zips', 'k8s'])
```

Comment:
nit: Would it be more organized to put `k8s` before `zips` to keep them in an alphabetic order?